pilot/pkg/config/coredatamodel/controller_test.go

```diff
@@ -179,7 +179,6 @@ func TestOptions(t *testing.T) {
 	change := convertToChange(
 		[]proto.Message{message},
 		[]string{"service-bar"},
-		setVersion("1"),
```

Comment:
why are you removing versions?

Reply:
These were not versions as in annotation versions. They were simply protocol(API) version which is not really needed nor needs to be tested in the unit tests, it just clutters the test.